cfn-resources/cluster/mongodb-atlas-cluster.json

```diff
@@ -16,60 +17,110 @@
         },
         "compute": {
             "type": "object",
+            "description": "Automatic Compute Scaling",
             "properties": {
                 "Enabled": {
-                    "type": "boolean"
+                    "type": "boolean",
+                    "description": "Flag that indicates whether someone enabled instance size auto-scaling.\n\nSet to true to enable instance size auto-scaling. If enabled, you must specify a value for replicationSpecs[n].regionConfigs[m].autoScaling.compute.maxInstanceSize.\nSet to false to disable instance size automatic scaling."
                 },
                 "ScaleDownEnabled": {
-                    "type": "boolean"
+                    "type": "boolean",
+                    "description": "Flag that indicates whether the instance size may scale down. MongoDB Cloud requires this parameter if \"replicationSpecs[n].regionConfigs[m].autoScaling.compute.enabled\" : true. If you enable this option, specify a value for replicationSpecs[n].regionConfigs[m].autoScaling.compute.minInstanceSize."
                 },
                 "MinInstanceSize": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Minimum instance size to which your cluster can automatically scale. MongoDB Cloud requires this parameter if \"replicationSpecs[n].regionConfigs[m].autoScaling.compute.enabled\" : true."
                 },
                 "MaxInstanceSize": {
-                    "type": "string"
+                    "type": "string",
+                    "description": "Maximum instance size to which your cluster can automatically scale. MongoDB Cloud requires this parameter if \"replicationSpecs[n].regionConfigs[m].autoScaling.compute.enabled\" : true."
                 }
             },
             "additionalProperties": false
         },
-        "regionsConfig": {
+        "AdvancedRegionConfig": {
```

Comment:
we are missing, capital and lower case definitions, we should define one standard to all